env0/utils.go

```diff
@@ -107,23 +106,9 @@ func readResourceData(i interface{}, d *schema.ResourceData) error {
 				return fmt.Errorf("internal error - unhandled field pointer kind %v", fieldType.Elem().Kind())
 			}
 		case reflect.Slice:
-			switch fieldType {
```

Comment:
This now has a generic solution. see below.

Reply:
The generic solution now works for most structs.
In this case, I needed support for this:
```
type TemplateSshKey struct {
	Id   string `json:"id"`
	Name string `json:"name"`
}

```